fortitude/tests/check.rs

```diff
@@ -851,3 +852,244 @@ per-file-ignores = [
 
     Ok(())
 }
+
+fn exclude_test_path<P: AsRef<Path>>(tempdir: P) -> PathBuf {
+    let base_path = tempdir.as_ref().join("base");
+    let foo_path = base_path.join("foo");
+    let bar_path = foo_path.join("bar");
+    // Simulate a Python env, which is in the default exclude list
+    let venv_path = base_path.join(".venv/lib/site-packages/numpy");
+    std::fs::create_dir_all(bar_path.as_path()).unwrap();
+    std::fs::create_dir_all(venv_path.as_path()).unwrap();
+    for dir in [&base_path, &foo_path, &bar_path, &venv_path] {
+        let name = dir.file_name().unwrap().to_string_lossy();
+        println!("{name}");
+        let snippet = format!(
+            r#"
+module {name}
+! missing implicit none
+contains
+  integer function f()
+    f = 1
+  end function f
+end module {name}
+"#
+        );
+        fs::write(dir.join(format!("{name}.f90")), snippet).unwrap();
+    }
+
+    let config_file = base_path.join(".fortitude.toml");
+    let config = r#"
+[check]
+exclude = [
+    "foo.f90",
+]
+"#;
+    fs::write(&config_file, config).unwrap();
+    base_path
+}
+
+#[test]
+fn check_exclude() -> anyhow::Result<()> {
+    let tempdir = TempDir::new()?;
+    apply_common_filters!();
+    // Expect:
+    // - Overwrite 'foo.f90' in config file, see 'base.f90' and 'foo.f90' but not 'bar.f90'
```

Comment:
Lowest possible priority, but you meant this?

```suggestion
    // - Override 'foo.f90' in config file, see 'base.f90' and 'foo.f90' but not 'bar.f90'
```